sphinxcontrib_verilog_diagrams.py

```diff
@@ -86,6 +87,23 @@ def verilog_diagram_name(srcpath, srclineno, verilog_path):
     )
 
 
+class NoLicenseInclude(LiteralInclude):
+    def run(self):
+        # type: () -> List[nodes.Node]
+
+        rel_filename, filename = self.env.relfn2path(self.arguments[0])
+        code = open(rel_filename, 'r').read().strip().split('\n')
+        first_line = next(
+            (idx + 3 for idx, line in enumerate(code) if 'SPDX' in line), 1)
```

Comment:
Currently, this assumes that the license header is in the C-style multiline comment format, with a newline after the comment, hence the `idx + 3`. For example

```
/*
 * Copyright (C) 2020  The SymbiFlow Authors.
 *
 * Use of this source code is governed by a ISC-style
 * license that can be found in the LICENSE file or at
 * https://opensource.org/licenses/ISC
 *
 * SPDX-License-Identifier:	ISC
 */

```

In the python files there will not be an extra line ending the comment (the ` */` line), and might not have a newline after the header (for example I see this in https://github.com/SymbiFlow/fasm/pull/21/commits/5d25dc781149aaab534bdc119130b5253cabf6e6). Since this is the verilog-diagrams extension, should we for now just narrow our use case to our verilog code which supposedly fits these assumptions, and not worry about the other languages (for example python)?

Reply:
Verilog supports both `/*` and `//` style comments IIRC, so we should at least support them.